### ISSUE TITLE
Why do you use gt of unlabeled data, "gt_mask_u_w" and "gt_mask_u_w_mix" for training?

### ISSUE THREAD

**Issue by djh-dzxw** [cvlab-stonybrook/Beyond-Pixels#5]
We noticed that "gt_mask_u_w" and "gt_mask_u_w_mix" affected the final loss function. How do you explain this? As far as we know, semi-supervised semantic segmentation tasks cannot use annotations with unlabeled data!

**Comment by prantikhowlader** [comment]
gt_mask_u_w and gt_mask_u_w_mix are used to generate mask_u_s1  and  mask_u_s2, this is used in compute_batch_loss (util/new_multilabel.py). Where we used it to calculate True positive and false positive. To evaluate our multilabel classifier. It is not used in any loss and backpropagated
